src/platform/ESP32/nimble/ChipDeviceScanner.h

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
There's already `src/platform/ESP32/ChipDeviceScanner.h` this may be left out when you removed one from bluedroid, can you please check.